README.md

```diff
@@ -1,358 +1,106 @@
 
-JSON:
-```
-"CORE_RPC": {
-    "HOST": "127.0.0.1",
-    "PORT": 8332,
-    "USERNAME": "mempool",
-    "PASSWORD": "mempool"
-  },
-```
-docker-compose overrides:
-```
-      CORE_RPC_HOST: ""
-      CORE_RPC_PORT: ""
-      CORE_RPC_USERNAME: ""
-      CORE_RPC_PASSWORD: ""
-```
+We support the following installation methods, ranked in order from simple to advanced:
 
-JSON:
-```
-  "ELECTRUM": {
-    "HOST": "127.0.0.1",
-    "PORT": 50002,
-    "TLS_ENABLED": true
-  },
-```
+1) [One-click installation on full-node distros](#one-click-installation)
+2) [Docker installation on Linux using docker-compose](./docker)
+3) [Manual installation on Linux or FreeBSD](#manual-installation)
+4) [Production installation on a powerful FreeBSD server](./production)
 
-docker-compose overrides:
-```
-      ELECTRUM_HOST: ""
-      ELECTRUM_PORT: ""
-      ELECTRUM_TLS: ""
-```
+This doc offers install notes on the one-click method and manual install method. Follow the links above for install notes on Docker and production installations.
 
-JSON:
-```
-  "ESPLORA": {
-    "REST_API_URL": "http://127.0.0.1:3000"
-  },
-```
-docker-compose overrides:
-```
-      ESPLORA_REST_API_URL: ""
-```
+<a id="one-click-installation"></a>
+## One-Click Installation
 
-JSON:
-```
-  "SECOND_CORE_RPC": {
-    "HOST": "127.0.0.1",
-    "PORT": 8332,
-    "USERNAME": "mempool",
-    "PASSWORD": "mempool"
-  },
-```
+Mempool can be conveniently installed on the following full-node distros: 
+- [Umbrel](https://github.com/getumbrel/umbrel)
+- [RaspiBlitz](https://github.com/rootzoll/raspiblitz)
+- [RoninDojo](https://code.samourai.io/ronindojo/RoninDojo)
+- [myNode](https://github.com/mynodebtc/mynode)
+- [Start9](https://github.com/Start9Labs/embassy-os)
 
-docker-compose overrides:
-```
-      SECOND_CORE_RPC_HOST: ""
-      SECOND_CORE_RPC_PORT: ""
-      SECOND_CORE_RPC_USERNAME: ""
-      SECOND_CORE_RPC_PASSWORD: ""
-```
+<a id="manual-installation"></a>
+## Manual Installation
 
-JSON:
-```
-  "DATABASE": {
-    "ENABLED": true,
-    "HOST": "127.0.0.1",
-    "PORT": 3306,
-    "DATABASE": "mempool",
-    "USERNAME": "mempool",
-    "PASSWORD": "mempool"
-  },
-```
+The following instructions are for a manual installation on Linux or FreeBSD. You may need to change file and directory paths to match your OS.
 
-docker-compose overrides:
-```
-      DATABASE_ENABLED: ""
-      DATABASE_HOST: ""
-      DATABASE_PORT: ""
-      DATABASE_DATABASE: ""
-      DATABASE_USERAME: ""
-      DATABASE_PASSWORD: ""
-```
+You will need [Bitcoin](https://github.com/bitcoin/bitcoin), [Electrum](https://github.com/romanz/electrs), [Node.js](https://github.com/nodejs/node), [MariaDB](https://github.com/mariadb/server), and [Nginx](https://github.com/nginx/nginx). Below, we walk through how to configure each of these.
```

Comment:
> `[Electrum](https://github.com/romanz/electrs)`

Electrum is a wallet, electrs is an electrum server implementation like ElectrumX